vcluster/configure/vcluster-yaml/sync/from-host/custom-resources.mdx

```diff
@@ -12,31 +12,67 @@ import NamespacedCustomResourcesExample from '../../../../_fragments/sync-from-h
+# Sync custom resources from host to virtual cluster
+
+vCluster allows you to sync custom resources from the host cluster to the virtual cluster. This feature creates read-only copies of custom resources inside your virtual cluster, making them available to applications and users without giving them direct access to the host cluster.
+
+When you enable custom resource syncing, vCluster first copies the CustomResourceDefinition (CRD) from the host to the virtual cluster, then begins syncing the actual custom resource instances. This is particularly useful for sharing cluster-wide resources like `ClusterIssuers` from [cert-manager](https://cert-manager.io/) or `ClusterStores` from [external-secrets](https://external-secrets.io/latest/) with your virtual cluster workloads.
+
+The synced resources are read-only in the virtual cluster. If you modify them in the host cluster, vCluster automatically updates the virtual cluster copies. However, changes made to these resources within the virtual cluster are not persisted.
+
+If you need to sync resources from the virtual cluster to the host cluster instead, see [syncing custom resources to the host cluster](../to-host/advanced/custom-resources).
 
 :::info No need to configure RBAC
 vCluster automatically adds the required cluster RBAC permissions for retrieving the CustomResourceDefinition and syncing the resources from the host to the virtual cluster.
 :::
 
-## Cluster scoped example
+## Cluster-scoped example
 
-To configure vCluster to sync ClusterIssuers from the host cluster (from [cert-manager](https://cert-manager.io/)):
-```yaml title="configure Cluster scoped CRD sync from host"
+Cluster-scoped custom resources exist at the cluster level and are not tied to any specific namespace. These are the simplest to sync because they don't require namespace mapping.
+
+To sync `ClusterIssuers` from the host cluster using [cert-manager](https://cert-manager.io/), configure the resource in your `vcluster.yaml` file:
+
+```yaml title="configure cluster-scoped CRD sync from host"
 sync:
   fromHost:
     customResources:
       clusterissuers.cert-manager.io:
         enabled: true
         scope: Cluster
 ```
-<!-- vale Google.Headings = NO -->
-## Namespace scoped CustomResourceDefinitions
-<!-- vale Google.Headings = YES -->
-By default, this is turned off.
 
-Enabling this allow you to sync namespaced CustomResources from the specified namespaces in the host to the specified namespaces in the vCluster.
-```yaml title="configure Namespace scoped CRD sync from host"
+By default, vCluster does not require you to specify an API version when syncing custom resources. When a CRD has multiple versions and you don't specify a version, vCluster uses the storage version.
+
+If you want to sync a specific version of a CRD that is not the storage version, you can explicitly specify it in the sync configuration.
```

Comment:
Should also write that this version is then `storage` in the virtual cluster, and be aware that CR will be converted between the two versions.

Reply:
Added